environment.yml

```diff
@@ -36,6 +36,7 @@ dependencies:
       - amazon-braket-sdk
       - dask==2.30.0
       - dwave-ocean-sdk==3.3.0
+      - ecos==2.0.7.post1
```

Comment:
We should also pin cvxpy. Also can we remove the pip install from the randomness generation notebook then, since it won't be needed ?

Reply:
Yes, let me check if that causes any issues with other dependencies.